include/tools.h

```diff
@@ -216,5 +216,14 @@ std::map<std::string, std::string> getNetworkInterfaces();
  */
 std::string getBestPublicIp();
 
+/** Converts file size to human readable format
+ * 
+ * This function will convert a number to its equivalent size using units.
+ * 
+ * @param file size in bytes
```

Comment:
The first word after `@param` must be the parameter name. So:
```
@param number file size in bytes.
```

And finish the sentences (even param and return descriptions) with `.`